traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -926,7 +926,7 @@ JOIN server_profile sp ON s.id = sp.server`
 		if err != nil {
 			return nil, serverCount, nil, fmt.Errorf("getting servers: %w", err), http.StatusInternalServerError, nil
 		}
-		if (version.GreaterThanOrEqualTo(&api.Version{Major: 4}) && roleBasedPerms) || version.GreaterThanOrEqualTo(&api.Version{Major: 5}) {
+		if (!version.LessThan(api.Version{Major: 4}) && roleBasedPerms) || !version.LessThan(api.Version{Major: 5}) {
```

Comment:
This line is particularly harder to reason about without `Version.GreaterThanOrEqualTo`, please change it back to that.